applications/GeoMechanicsApplication/custom_conditions/micro_climate_constants.h

```diff
@@ -0,0 +1,34 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#pragma once
+
+namespace Kratos::MicroClimateConstants
```

Comment:
Wise to have 1 place for definitions.

They seem a mix of material parameters and physical constants. Sadly some of them are redefinitions of already existing parameters.

Reply:
We removed the redefinition of the water density and as seen in the other comment, the gravity constant is a bigger issue that we'd like to tackle seperately